src/megaclient.cpp

```diff
@@ -2996,11 +2963,9 @@ bool MegaClient::procsc()
                                 // node update
                                 sc_updatenode();
 #ifdef ENABLE_SYNC
-                                if (!fetchingnodes && jsonsc.pos[1] != ']') // there are more packets
-                                {
-                                    applykeys();
-                                    return false;
-                                }
+                                // run syncdown() before continuing
```

Comment:
We must control that the AP is not part of a cached fetchnodes. If so, do not return false (do not force any sync scan)